src/Windows/Perspex.Direct2D1/PrimitiveExtensions.cs

```diff
@@ -5,25 +5,50 @@
 using System.Linq;
 using SharpDX;
 using SharpDX.Direct2D1;
+using SharpDX.Mathematics.Interop;
 using DWrite = SharpDX.DirectWrite;
 
 namespace Perspex.Direct2D1
 {
     public static class PrimitiveExtensions
     {
-        public static Rect ToPerspex(this RectangleF r)
+        /// <summary>
+        /// The value for which all absolute numbers smaller than are considered equal to zero.
+        /// </summary>
+        public const float ZeroTolerance = 1e-6f; // Value a 8x higher than 1.19209290E-07F
```

Comment:
This line doesn't seem to be related to the SharpDX 3 port. Has it crept in from somewhere else?